### PR TITLE
Update instructions to always create the image in the user's project.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

File: vm-images/README.md [hunk 1]
```diff
@@ -3,32 +3,21 @@ Google Compute Engine virtual machine images
...
+Scripts that create a Google Compute Engine image containing a set of popular genomics software. The exact list of software available will change with time, but examples of software it may contain are [TopHat](https://ccb.jhu.edu/software/tophat/index.shtml), [bwa](http://bio-bwa.sourceforge.net/), and [freebayes](https://github.com/ekg/freebayes).  Additionally, R is pre-installed along with some [Bioconductor](http://www.bioconductor.org/) packages and [bigrquery](http://cran.r-project.org/web/packages/bigrquery/index.html), the R interface to Google BigQuery.  See [omics_startup_script.sh] for the current list of
```
Comment (mbookman) [inline vm-images/README.md]: Would you mind putting some line breaks into this paragraph?
Not to change the rendered formatting, but to change the underlying source.
It was very hard to figure out the change here.

File: vm-images/README.md [hunk 2]
```diff
@@ -3,32 +3,21 @@ Google Compute Engine virtual machine images
...
-from the [public space]
-(https://console.developers.google.com/storage/genomics-public-data/gce-images/)
+## Create the Image
 
+To create a new image directly in a Google Cloud Platform project, execute the shell script:
 ```
-  gcloud compute images create omics-<id> --project <project-name> --source-uri gs://genomics-public-data/gce-images/omics-<id>.image.tar.gz
+./omics_create_image.sh <YOUR-PROJECT-ID> gs://<YOUR-BUCKET>/<OPTIONAL-SUBDIRECTORY>
 ```
+After about an hour you will see the new `.tar.gz` file under the Cloud Storage path specified and the virtual machine will be automatically deleted..
```
Comment (mbookman) [inline vm-images/README.md]: "and the virtual machine will be automatically deleted". What virtual machine?

**Comment by deflaux** [comment]
PTAL, thanks!

**Comment by mbookman** [comment]
LGTM

**Pull request merged**